### DISCUSSION TITLE
Steps after cloning a repo on my new pc

### DISCUSSION THREAD

**Discussion by EmaProv** [community/community#24024] [New to GitHub]
<p>Hi everyone,<br>
I’m new in this world and starting to learn the basic of github. I have a question: what are the basic step I have to do when I clone my project repo on a new device? I’m working on a project with React JS and Node JS, and I push everything on my private repo in my profile. Now when I clone my repo in my new pc, the basic comand as ‘npm start’ doesn’t work. Am I missing some step?</p>
<p>NOTE: I’m noticed in that my gitignore file doesn’t make me push the node_module folder where I install all my dependencies, can this cause some problem to the correct execution of my coned proj?</p>
<p>thanks for your time.<br>
E.P.</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/emaprov">@EmaProv</a> :wave:</p>
<p>Try installing your dependencies from your <code>package.json</code> file.</p>
<p>Run <code>npm install</code><br>
That should look to your package file for all the node modules that need to be installed.</p>
<p>Also, it would be helpful if you posted the <code>scripts</code> portion of your <code>package.json</code> file. This will help us see what your start script says to do.</p>

**Comment by EmaProv** [comment]
<p>Thanks for your help! I had some issue because I was trying to clone my repo from Win to MacOS, and the zsh bash throws me some errors. Cloning from Win to Win I had no problem!</p>

**Reply by okekolawolesunday009** [reply]
helpful
